### PR TITLE
Merge upstream

### PR DESCRIPTION
**Pull request by trombik**
update `reallyenglish-master` to the latest upstream master.

this PR is for `reallyenglish-master`.

### CODE REVIEW

File: debian-9.0-amd64.json [hunk 1]
```diff
@@ -27,20 +28,20 @@
   }, {
     "type": "virtualbox-iso",
     "guest_os_type": "Debian_64",
-    "iso_url": "{{user `mirror`}}/8.7.1/amd64/iso-cd/debian-8.7.1-amd64-netinst.iso",
+    "iso_url": "{{user `mirror`}}/9.0.0/amd64/iso-cd/debian-9.0.0-amd64-netinst.iso",
     "iso_checksum": "{{user `iso_checksum`}}",
     "iso_checksum_type": "{{user `iso_checksum_type`}}",
-    "output_directory": "output-debian-8.7-amd64-{{build_type}}",
-    "vm_name": "packer-debian-8.7-amd64",
-    "hard_drive_interface": "scsi",
```
Comment (mitsururike) [inline debian-9.0-amd64.json]: this must be merged.

File: http/debian-9.0/preseed.cfg [hunk 2]
```diff
@@ -52,10 +52,7 @@ d-i preseed/early_command string
         -e "/in-target/i echo 'd() { /sbin/discover \"\$@\" | grep -v virtualbox; }' > /target/etc/discover-pkginstall.conf"  \
         -e "/in-target/i echo 'discover=d' >> /target/etc/discover-pkginstall.conf"                                           \
         /usr/lib/pre-pkgsel.d/20install-hwpackages
-
 d-i preseed/late_command string                                                   \
         echo 'Defaults:vagrant !requiretty' > /target/etc/sudoers.d/vagrant;      \
         echo 'vagrant ALL=(ALL) NOPASSWD: ALL' >> /target/etc/sudoers.d/vagrant;  \
-        chmod 440 /target/etc/sudoers.d/vagrant;                                  \
-        sed -i -e 's/^GRUB_TIMEOUT=.*$/GRUB_TIMEOUT=0/' /target/etc/default/grub; \
```
Comment (mitsururike) [inline http/debian-9.0/preseed.cfg]: this must be merged.

**Review by mitsururike — COMMENTED** [review]
debian-8.7 is used in https://github.com/reallyenglish/ansible-role-java.

**Comment by trombik** [comment]
we really need a proper procedure to merge upstream.

**Comment by trombik** [comment]
there might be missing test cases. please test by following the instruction described in c1c2c9d64ea7a5c6900717c81b8225bf51d65511

the test takes a long time, probably more than an hour.

**Comment by mitsururike** [comment]
failed to build debian-9.0
```
    virtualbox-iso: + sudo apt-get -y -t jessie-backports install ansible
    virtualbox-iso: Reading package lists...
    virtualbox-iso: E: The value 'jessie-backports' is invalid for APT::Default-Release as such a release is not available in the sources
```

**Comment by trombik** [comment]
fixed in a8a552a

**Review by mitsururike — APPROVED** [review]

**Pull request merged**